mava/evaluator.py

```diff
@@ -105,7 +105,9 @@ def evaluator_fn(trained_params: FrozenDict, rng: chex.PRNGKey) -> ExperimentOut
         # Add dimension to pmap over.
         step_rngs = jnp.stack(step_rngs).reshape(eval_batch, -1)
 
-        eval_state = EvalState(step_rngs, env_states, timesteps, 0, 0.0)
+        eval_state = EvalState(
+            step_rngs, env_states, timesteps, 0, jnp.zeros_like(timesteps.reward)
```

Comment:
Is this so that we can support team and individual rewards?

Reply:
Actually, this was changed because we removed the reshaping of the reward outside of the system here [https://github.com/instadeepai/Mava/blob/7e500aaa2c657ce818689c589020f39284391da4/mava/systems/ff_ippo_rware.py#L152] and added it the wrapper in the modify_timesteps [https://github.com/instadeepai/Mava/blob/7e500aaa2c657ce818689c589020f39284391da4/mava/wrappers/jumanji.py#L77C6-L77C6] so that in future we can use the same system.